pkg/render/fluentd_test.go

```diff
@@ -884,6 +895,15 @@ var _ = Describe("Tigera Secure Fluentd rendering tests", func() {
 			}))
 
 		Expect(envs).To(ContainElement(corev1.EnvVar{Name: "EKS_CLOUDWATCH_LOG_FETCH_INTERVAL", Value: "900"}))
+		Expect(envs).To(ContainElements([]corev1.EnvVar{
```

Comment:
Probably want the reverse check as well to catch any cases where there are unexpected env vars. And maybe a length check to catch bugs like the one above where we added env vars twice.

e.g.,

```go
expected := []corev1.EnvVar{...}

Expect(envs).Should(ContainElements(expected))
Expect(expected).Should(ContainElements(envs))
Expect(envs).Should(HaveLength(len(expected))
```

Also, we can merge the assertion on the line right above this.